www/js/init.js

```diff
@@ -123,6 +125,8 @@ params['cacheAPI'] = 'kiwix-js'; // Sets name of the prefix used to identify the
 params['cacheIDB'] = 'kiwix-zim'; // Sets name of the Indexed DB database
 params['isFileSystemApiSupported'] = typeof window.showOpenFilePicker === 'function'; // Sets a boolean indicating whether the FileSystem API is supported
 params['isWebkitDirApiSupported'] = 'webkitdirectory' in document.createElement('input'); // Sets a boolean indicating whether the Webkit Directory API is supported
+params['libzimMode'] = getSetting('libzimMode') ?? 'default'; // Sets a value indicating which libzim mode is selected
+params['useLibzim'] = params.libzimMode !== 'default'; // Sets a value indicating which libzim mode is selected
```

Comment:
So you don't have a separate checkbox setting for `params.useLibzim` in the UI? Wouldn't that be cleaner than depending on a potentially confusing side-effect here?

Reply:
I don't think we need an extra button to enable libzim since
- It will be enabled nonetheless (in case of default for full-text search)
- I added `useLibzim` only so that anyone knows the file read will be also done by libzim

I don't have a strong opinion here if you think I should directly use `params.libzimMode !== "default"`. I will make those changes

Reply:
OK, I'll check when I test the UI.